### PR TITLE
doc/github-actions: documented 'setup-atlas' action

### PR DESCRIPTION
**Pull request by dorav**

### CODE REVIEW

File: doc/md/integrations/github-actions.mdx [hunk 1]
```diff
@@ -379,4 +380,33 @@ Configure this action with the following inputs:
 
 #### `cloud-token`
 
-**Required** The Atlas Cloud token to use for authentication.
\ No newline at end of file
+**Required** The Atlas Cloud token to use for authentication.
+
+## `ariga/setup-atlas`
+
+The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install Atlas from a github action.
```
Comment (rotemtam) [inline doc/md/integrations/github-actions.mdx:387]:
```suggestion
The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install the Atlas CLI from a GitHub Actions workflow.
```

File: doc/md/integrations/github-actions.mdx [hunk 2]
```diff
@@ -379,4 +380,33 @@ Configure this action with the following inputs:
...
+      - uses: ariga/setup-atlas@master
+      - run: atlas version
+      # - run: atlas schema apply ... 
+```
+
+### Reference
+
+The action supports the `Version` input:
+
+#### `Version`
```
Comment (ronenlu) [inline doc/md/integrations/github-actions.mdx:410]: maybe give an example of the semantic versioning, such as:
`
with:
    Version: v0.13.2
`

File: doc/md/integrations/github-actions.mdx [hunk 3]
```diff
@@ -25,6 +25,7 @@ database schema management tasks.
 | [ariga/atlas-action](https://github.com/ariga/atlas-action)        | CI for schema changes                               |
 | [ariga/atlas-sync-action](https://github.com/ariga/atlas-sync-action)   | Sync your migration directory to Atlas Cloud (atlasgo.cloud) |
 | [ariga/atlas-deploy-action](https://github.com/ariga/atlas-deploy-action) | Deploy versioned migrations from GitHub Actions    |
+| [ariga/setup-atlas](https://github.com/ariga/set-uatlas) | Install Atlas from a Github Action    |
```
Comment (rotemtam) [inline doc/md/integrations/github-actions.mdx:28]:
```suggestion
| [ariga/setup-atlas](https://github.com/ariga/setup-atlas) | Install Atlas from a GitHub Actions workflow    |
```

File: doc/md/integrations/github-actions.mdx [hunk 4]
```diff
@@ -379,4 +380,33 @@ Configure this action with the following inputs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ariga/setup-atlas@master
+      - run: atlas version
+      # - run: atlas schema apply ... 
+```
+
+### Reference
+
+The action supports the `Version` input:
```
Comment (rotemtam) [inline doc/md/integrations/github-actions.mdx:408]: Action inputs are commonly `kebab-cased`
```suggestion
The action supports the `version` input:
```

File: doc/md/integrations/github-actions.mdx (hunk 2 above)
Comment (rotemtam) [inline doc/md/integrations/github-actions.mdx:410]:
```suggestion
#### `version`
```

**Review by rotemtam — CHANGES_REQUESTED** [review]

File: doc/md/integrations/github-actions.mdx (hunk 1 above)
Comment (dorav) [reply doc/md/integrations/github-actions.mdx:387]: In the previous PR I used `Atlas CLI` and Mashraki suggested to use just `Atlas`

Let's keep it consistent? which one should I use?

Comment (rotemtam) [reply doc/md/integrations/github-actions.mdx:387]: `Atlas` , sorry.

Comment (dorav) [reply doc/md/integrations/github-actions.mdx]: 👍 
Done

**Review by rotemtam — APPROVED** [review]

**Pull request merged**